### PR TITLE
python3Packages: fix some cross compilation

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/libversion/default.nix [hunk 1]
```diff
@@ -28,6 +34,15 @@ buildPythonPackage rec {
     libversion
   ];
 
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  preCheck = ''
```
Comment (Mindavi) [inline pkgs/development/python-modules/libversion/default.nix:41]:
```suggestion
  postBuild = ''
```

**Review by Mindavi — APPROVED** [review]

File: pkgs/development/python-modules/libversion/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/python-modules/libversion/default.nix:41]: I'm not sure tbh but I'm more comfortable with doing that after installation.

Comment (Mindavi) [reply pkgs/development/python-modules/libversion/default.nix:41]: all good then

**Review by jonringer — COMMENTED** [review]
if these aren't critical now, I would prefer to target staging. 500+ rebuilds

If you don't want to wait a whole staging cycle, then staging-next might be a good candidate as well.

**Comment by dotlambda** [comment]
> if these aren't critical now, I would prefer to target staging. 500+ rebuilds

The rebuilds are mainly Home Assistent tests, that's why I'm targeting master.

**Comment by jonringer** [comment]
sounds good then

**Review by jonringer — APPROVED** [review]
LGTM

failures are broken on target branch
```
https://github.com/NixOS/nixpkgs/pull/177591

13 packages marked as broken and skipped:
apache-airflow home-assistant-component-tests.bsblan home-assistant-component-tests.dnsip home-assistant-component-tests.efergy home-assistant-component-tests.ssdp home-assistant-component-tests.subaru lexicon python310Packages.apache-airflow python310Packages.hug python310Packages.pytest-sanic python39Packages.apache-airflow python39Packages.hug python39Packages.pytest-sanic

1 package failed to build:
home-assistant-component-tests.history_stats

858 packages built:
atomic-operator audible-cli beets beets-unstable borgbackup borgmatic cfripper crackmapexec datasette dmarc-metrics-exporter dyndnsc esphome etebase-server faraday-cli home-assistant home-assistant-component-tests.abode home-assistant-component-tests.accuweather home-assistant-component-tests.acmeda home-assistant-component-tests.adax home-assistant-component-tests.adguard home-assistant-component-tests.advantage_air home-assistant-component-tests.aemet home-assistant-component-tests.agent_dvr home-assistant-component-tests.air_quality home-assistant-component-tests.airly home-assistant-component-tests.airnow home-assistant-component-tests.airthings home-assistant-component-tests.airtouch4 home-assistant-component-tests.airvisual home-assistant-component-tests.airzone home-assistant-component-tests.aladdin_connect home-assistant-component-tests.alarm_control_panel home-assistant-component-tests.alarmdecoder home-assistant-component-tests.alert home-assistant-component-tests.alexa home-assistant-component-tests.almond home-assistant-component-tests.ambee home-assistant-component-tests.amberelectric home-assistant-component-tests.ambiclimate home-assistant-component-tests.ambient_station home-assistant-component-tests.analytics home-assistant-component-tests.androidtv home-assistant-component-tests.apache_kafka home-assistant-component-tests.api home-assistant-component-tests.apple_tv home-assistant-component-tests.application_credentials home-assistant-component-tests.apprise home-assistant-component-tests.aprs home-assistant-component-tests.arcam_fmj home-assistant-component-tests.aseko_pool_live home-assistant-component-tests.asuswrt home-assistant-component-tests.atag home-assistant-component-tests.august home-assistant-component-tests.aurora home-assistant-component-tests.aurora_abb_powerone home-assistant-component-tests.aussie_broadband home-assistant-component-tests.auth home-assistant-component-tests.automation home-assistant-component-tests.awair home-assistant-component-tests.aws home-assistant-component-tests.axis home-assistant-component-tests.azure_devops home-assistant-component-tests.azure_event_hub home-assistant-component-tests.backup home-assistant-component-tests.balboa home-assistant-component-tests.bayesian home-assistant-component-tests.binary_sensor home-assistant-component-tests.blackbird home-assistant-component-tests.blebox home-assistant-component-tests.blink home-assistant-component-tests.blueprint home-assistant-component-tests.bluetooth_le_tracker home-assistant-component-tests.bmw_connected_drive home-assistant-component-tests.bond home-assistant-component-tests.bosch_shc home-assistant-component-tests.braviatv home-assistant-component-tests.broadlink home-assistant-component-tests.brother home-assistant-component-tests.brunt home-assistant-component-tests.buienradar home-assistant-component-tests.button home-assistant-component-tests.caldav home-assistant-component-tests.calendar home-assistant-component-tests.camera home-assistant-component-tests.canary home-assistant-component-tests.cast home-assistant-component-tests.cert_expiry home-assistant-component-tests.climacell home-assistant-component-tests.climate home-assistant-component-tests.cloud home-assistant-component-tests.cloudflare home-assistant-component-tests.co2signal home-assistant-component-tests.color_extractor home-assistant-component-tests.comfoconnect home-assistant-component-tests.command_line home-assistant-component-tests.compensation home-assistant-component-tests.config home-assistant-component-tests.configurator home-assistant-component-tests.control4 home-assistant-component-tests.conversation home-assistant-component-tests.coolmaster home-assistant-component-tests.coronavirus home-assistant-component-tests.counter home-assistant-component-tests.cover home-assistant-component-tests.cpuspeed home-assistant-component-tests.crownstone home-assistant-component-tests.daikin home-assistant-component-tests.darksky home-assistant-component-tests.datadog home-assistant-component-tests.debugpy home-assistant-component-tests.deconz home-assistant-component-tests.default_config home-assistant-component-tests.deluge home-assistant-component-tests.demo home-assistant-component-tests.denonavr home-assistant-component-tests.derivative home-assistant-component-tests.device_automation home-assistant-component-tests.device_sun_light_trigger home-assistant-component-tests.device_tracker home-assistant-component-tests.devolo_home_control home-assistant-component-tests.devolo_home_network home-assistant-component-tests.dexcom home-assistant-component-tests.dhcp home-assistant-component-tests.diagnostics home-assistant-component-tests.dialogflow home-assistant-component-tests.directv home-assistant-component-tests.discord home-assistant-component-tests.discovery home-assistant-component-tests.dlna_dmr home-assistant-component-tests.dlna_dms home-assistant-component-tests.doorbird home-assistant-component-tests.dsmr home-assistant-component-tests.dte_energy_bridge home-assistant-component-tests.duckdns home-assistant-component-tests.dunehd home-assistant-component-tests.dynalite home-assistant-component-tests.eafm home-assistant-component-tests.ecobee home-assistant-component-tests.econet home-assistant-component-tests.elgato home-assistant-component-tests.elkm1 home-assistant-component-tests.elmax home-assistant-component-tests.emonitor home-assistant-component-tests.emulated_hue home-assistant-component-tests.emulated_kasa home-assistant-component-tests.emulated_roku home-assistant-component-tests.energy home-assistant-component-tests.enocean home-assistant-component-tests.enphase_envoy home-assistant-component-tests.environment_canada home-assistant-component-tests.epson home-assistant-component-tests.esphome home-assistant-component-tests.everlights home-assistant-component-tests.evil_genius_labs home-assistant-component-tests.ezviz home-assistant-component-tests.faa_delays home-assistant-component-tests.facebook home-assistant-component-tests.facebox home-assistant-component-tests.fail2ban home-assistant-component-tests.fan home-assistant-component-tests.feedreader home-assistant-component-tests.ffmpeg home-assistant-component-tests.fibaro home-assistant-component-tests.fido home-assistant-component-tests.file home-assistant-component-tests.filesize home-assistant-component-tests.filter home-assistant-component-tests.fireservicerota home-assistant-component-tests.firmata home-assistant-component-tests.fivem home-assistant-component-tests.fjaraskupan home-assistant-component-tests.flic home-assistant-component-tests.flick_electric home-assistant-component-tests.flipr home-assistant-component-tests.flo home-assistant-component-tests.flume home-assistant-component-tests.flunearyou home-assistant-component-tests.flux home-assistant-component-tests.flux_led home-assistant-component-tests.folder home-assistant-component-tests.folder_watcher home-assistant-component-tests.foobot home-assistant-component-tests.forecast_solar home-assistant-component-tests.foscam home-assistant-component-tests.freebox home-assistant-component-tests.freedns home-assistant-component-tests.freedompro home-assistant-component-tests.fritz home-assistant-component-tests.fritzbox home-assistant-component-tests.fritzbox_callmonitor home-assistant-component-tests.fronius home-assistant-component-tests.frontend home-assistant-component-tests.garages_amsterdam home-assistant-component-tests.gdacs home-assistant-component-tests.generic home-assistant-component-tests.generic_hygrostat home-assistant-component-tests.generic_thermostat home-assistant-component-tests.geo_json_events home-assistant-component-tests.geo_location home-assistant-component-tests.geo_rss_events home-assistant-component-tests.geocaching home-assistant-component-tests.geofency home-assistant-component-tests.geonetnz_quakes home-assistant-component-tests.geonetnz_volcano home-assistant-component-tests.gios home-assistant-component-tests.github home-assistant-component-tests.glances home-assistant-component-tests.goalzero home-assistant-component-tests.gogogate2 home-assistant-component-tests.goodwe home-assistant-component-tests.google home-assistant-component-tests.google_assistant home-assistant-component-tests.google_domains home-assistant-component-tests.google_pubsub home-assistant-component-tests.google_translate home-assistant-component-tests.google_travel_time home-assistant-component-tests.google_wifi home-assistant-component-tests.gpslogger home-assistant-component-tests.graphite home-assistant-component-tests.gree home-assistant-component-tests.greeneye_monitor home-assistant-component-tests.group home-assistant-component-tests.growatt_server home-assistant-component-tests.guardian home-assistant-component-tests.habitica home-assistant-component-tests.hangouts home-assistant-component-tests.hardkernel home-assistant-component-tests.hardware home-assistant-component-tests.harmony home-assistant-component-tests.hassio home-assistant-component-tests.hddtemp home-assistant-component-tests.heos home-assistant-component-tests.here_travel_time home-assistant-component-tests.hisense_aehw4a1 home-assistant-component-tests.history home-assistant-component-tests.hive home-assistant-component-tests.hlk_sw16 home-assistant-component-tests.home_connect home-assistant-component-tests.home_plus_control home-assistant-component-tests.homeassistant home-assistant-component-tests.homekit home-assistant-component-tests.homekit_controller home-assistant-component-tests.homematic home-assistant-component-tests.homematicip_cloud home-assistant-component-tests.homewizard home-assistant-component-tests.honeywell home-assistant-component-tests.html5 home-assistant-component-tests.http home-assistant-component-tests.huawei_lte home-assistant-component-tests.hue home-assistant-component-tests.huisbaasje home-assistant-component-tests.humidifier home-assistant-component-tests.hunterdouglas_powerview home-assistant-component-tests.hvv_departures home-assistant-component-tests.hyperion home-assistant-component-tests.ialarm home-assistant-component-tests.ialarm_xr home-assistant-component-tests.iaqualink home-assistant-component-tests.icloud home-assistant-component-tests.ifttt home-assistant-component-tests.ign_sismologia home-assistant-component-tests.image home-assistant-component-tests.image_processing home-assistant-component-tests.imap_email_content home-assistant-component-tests.influxdb home-assistant-component-tests.input_boolean home-assistant-component-tests.input_button home-assistant-component-tests.input_datetime home-assistant-component-tests.input_number home-assistant-component-tests.input_select home-assistant-component-tests.input_text home-assistant-component-tests.insteon home-assistant-component-tests.integration home-assistant-component-tests.intellifire home-assistant-component-tests.intent home-assistant-component-tests.intent_script home-assistant-component-tests.ios home-assistant-component-tests.iotawatt home-assistant-component-tests.ipma home-assistant-component-tests.ipp home-assistant-component-tests.iqvia home-assistant-component-tests.islamic_prayer_times home-assistant-component-tests.isy994 home-assistant-component-tests.izone home-assistant-component-tests.jellyfin home-assistant-component-tests.jewish_calendar home-assistant-component-tests.juicenet home-assistant-component-tests.keenetic_ndms2 home-assistant-component-tests.kira home-assistant-component-tests.kmtronic home-assistant-component-tests.knx home-assistant-component-tests.kodi home-assistant-component-tests.konnected home-assistant-component-tests.kraken home-assistant-component-tests.kulersky home-assistant-component-tests.lastfm home-assistant-component-tests.launch_library home-assistant-component-tests.laundrify home-assistant-component-tests.lcn home-assistant-component-tests.light home-assistant-component-tests.litterrobot home-assistant-component-tests.local_file home-assistant-component-tests.local_ip home-assistant-component-tests.locative home-assistant-component-tests.lock home-assistant-component-tests.logbook home-assistant-component-tests.logentries home-assistant-component-tests.logger home-assistant-component-tests.logi_circle home-assistant-component-tests.london_air home-assistant-component-tests.lookin home-assistant-component-tests.lovelace home-assistant-component-tests.luftdaten home-assistant-component-tests.lutron_caseta home-assistant-component-tests.lyric home-assistant-component-tests.mailbox home-assistant-component-tests.mailgun home-assistant-component-tests.manual home-assistant-component-tests.manual_mqtt home-assistant-component-tests.maxcube home-assistant-component-tests.mazda home-assistant-component-tests.meater home-assistant-component-tests.media_player home-assistant-component-tests.media_source home-assistant-component-tests.melcloud home-assistant-component-tests.meraki home-assistant-component-tests.met home-assistant-component-tests.met_eireann home-assistant-component-tests.meteo_france home-assistant-component-tests.meteoclimatic home-assistant-component-tests.metoffice home-assistant-component-tests.microsoft_face home-assistant-component-tests.microsoft_face_detect home-assistant-component-tests.microsoft_face_identify home-assistant-component-tests.mikrotik home-assistant-component-tests.mill home-assistant-component-tests.min_max home-assistant-component-tests.minecraft_server home-assistant-component-tests.minio home-assistant-component-tests.mjpeg home-assistant-component-tests.mobile_app home-assistant-component-tests.modbus home-assistant-component-tests.modem_callerid home-assistant-component-tests.modern_forms home-assistant-component-tests.mold_indicator home-assistant-component-tests.moon home-assistant-component-tests.motion_blinds home-assistant-component-tests.motioneye home-assistant-component-tests.mqtt home-assistant-component-tests.mqtt_eventstream home-assistant-component-tests.mqtt_json home-assistant-component-tests.mqtt_room home-assistant-component-tests.mqtt_statestream home-assistant-component-tests.mullvad home-assistant-component-tests.mutesync home-assistant-component-tests.my home-assistant-component-tests.myq home-assistant-component-tests.mysensors home-assistant-component-tests.mythicbeastsdns home-assistant-component-tests.nam home-assistant-component-tests.namecheapdns home-assistant-component-tests.nanoleaf home-assistant-component-tests.neato home-assistant-component-tests.ness_alarm home-assistant-component-tests.nest home-assistant-component-tests.netatmo home-assistant-component-tests.netgear home-assistant-component-tests.network home-assistant-component-tests.nexia home-assistant-component-tests.nextbus home-assistant-component-tests.nightscout home-assistant-component-tests.nina home-assistant-component-tests.no_ip home-assistant-component-tests.notify home-assistant-component-tests.notify_events home-assistant-component-tests.notion home-assistant-component-tests.nsw_rural_fire_service_feed home-assistant-component-tests.nuki home-assistant-component-tests.number home-assistant-component-tests.nut home-assistant-component-tests.nws home-assistant-component-tests.nx584 home-assistant-component-tests.octoprint home-assistant-component-tests.omnilogic home-assistant-component-tests.onboarding home-assistant-component-tests.oncue home-assistant-component-tests.ondilo_ico home-assistant-component-tests.onewire home-assistant-component-tests.onvif home-assistant-component-tests.open_meteo home-assistant-component-tests.openalpr_cloud home-assistant-component-tests.openalpr_local home-assistant-component-tests.openerz home-assistant-component-tests.opengarage home-assistant-component-tests.openhardwaremonitor home-assistant-component-tests.opentherm_gw home-assistant-component-tests.openuv home-assistant-component-tests.openweathermap home-assistant-component-tests.opnsense home-assistant-component-tests.overkiz home-assistant-component-tests.ovo_energy home-assistant-component-tests.owntracks home-assistant-component-tests.p1_monitor home-assistant-component-tests.panasonic_viera home-assistant-component-tests.panel_custom home-assistant-component-tests.panel_iframe home-assistant-component-tests.peco home-assistant-component-tests.persistent_notification home-assistant-component-tests.person home-assistant-component-tests.philips_js home-assistant-component-tests.pi_hole home-assistant-component-tests.picnic home-assistant-component-tests.ping home-assistant-component-tests.plaato home-assistant-component-tests.plant home-assistant-component-tests.plex home-assistant-component-tests.plugwise home-assistant-component-tests.point home-assistant-component-tests.poolsense home-assistant-component-tests.powerwall home-assistant-component-tests.profiler home-assistant-component-tests.prometheus home-assistant-component-tests.prosegur home-assistant-component-tests.proximity home-assistant-component-tests.pure_energie home-assistant-component-tests.push home-assistant-component-tests.pushbullet home-assistant-component-tests.pvoutput home-assistant-component-tests.pvpc_hourly_pricing home-assistant-component-tests.python_script home-assistant-component-tests.qld_bushfire home-assistant-component-tests.qnap_qsw home-assistant-component-tests.rachio home-assistant-component-tests.radarr home-assistant-component-tests.radio_browser home-assistant-component-tests.rainforest_eagle home-assistant-component-tests.rainmachine home-assistant-component-tests.random home-assistant-component-tests.raspberry_pi home-assistant-component-tests.rdw home-assistant-component-tests.recollect_waste home-assistant-component-tests.recorder home-assistant-component-tests.reddit home-assistant-component-tests.remote home-assistant-component-tests.renault home-assistant-component-tests.rest home-assistant-component-tests.rest_command home-assistant-component-tests.rflink home-assistant-component-tests.rfxtrx home-assistant-component-tests.ridwell home-assistant-component-tests.ring home-assistant-component-tests.risco home-assistant-component-tests.rituals_perfume_genie home-assistant-component-tests.rmvtransport home-assistant-component-tests.roku home-assistant-component-tests.roomba home-assistant-component-tests.roon home-assistant-component-tests.rpi_power home-assistant-component-tests.rss_feed_template home-assistant-component-tests.rtsp_to_webrtc home-assistant-component-tests.ruckus_unleashed home-assistant-component-tests.sabnzbd home-assistant-component-tests.safe_mode home-assistant-component-tests.samsungtv home-assistant-component-tests.scene home-assistant-component-tests.scrape home-assistant-component-tests.screenlogic home-assistant-component-tests.script home-assistant-component-tests.search home-assistant-component-tests.season home-assistant-component-tests.select home-assistant-component-tests.sense home-assistant-component-tests.senseme home-assistant-component-tests.sensibo home-assistant-component-tests.sensor home-assistant-component-tests.sentry home-assistant-component-tests.senz home-assistant-component-tests.seventeentrack home-assistant-component-tests.sharkiq home-assistant-component-tests.shell_command home-assistant-component-tests.shelly home-assistant-component-tests.shopping_list home-assistant-component-tests.sia home-assistant-component-tests.sigfox home-assistant-component-tests.sighthound home-assistant-component-tests.signal_messenger home-assistant-component-tests.simplisafe home-assistant-component-tests.simulated home-assistant-component-tests.siren home-assistant-component-tests.slack home-assistant-component-tests.sleepiq home-assistant-component-tests.slimproto home-assistant-component-tests.sma home-assistant-component-tests.smappee home-assistant-component-tests.smart_meter_texas home-assistant-component-tests.smartthings home-assistant-component-tests.smarttub home-assistant-component-tests.smhi home-assistant-component-tests.smtp home-assistant-component-tests.snips home-assistant-component-tests.solaredge home-assistant-component-tests.solarlog home-assistant-component-tests.solax home-assistant-component-tests.soma home-assistant-component-tests.somfy home-assistant-component-tests.somfy_mylink home-assistant-component-tests.sonarr home-assistant-component-tests.songpal home-assistant-component-tests.sonos home-assistant-component-tests.soundtouch home-assistant-component-tests.spaceapi home-assistant-component-tests.spc home-assistant-component-tests.speedtestdotnet home-assistant-component-tests.spider home-assistant-component-tests.spotify home-assistant-component-tests.sql home-assistant-component-tests.squeezebox home-assistant-component-tests.srp_energy home-assistant-component-tests.starline home-assistant-component-tests.startca home-assistant-component-tests.statistics home-assistant-component-tests.statsd home-assistant-component-tests.steam_online home-assistant-component-tests.steamist home-assistant-component-tests.stookalert home-assistant-component-tests.stream home-assistant-component-tests.stt home-assistant-component-tests.sun home-assistant-component-tests.surepetcare home-assistant-component-tests.switch home-assistant-component-tests.switch_as_x home-assistant-component-tests.switchbot home-assistant-component-tests.switcher_kis home-assistant-component-tests.syncthing home-assistant-component-tests.syncthru home-assistant-component-tests.synology_dsm home-assistant-component-tests.system_bridge home-assistant-component-tests.system_health home-assistant-component-tests.system_log home-assistant-component-tests.tado home-assistant-component-tests.tag home-assistant-component-tests.tailscale home-assistant-component-tests.tankerkoenig home-assistant-component-tests.tasmota home-assistant-component-tests.tautulli home-assistant-component-tests.tcp home-assistant-component-tests.telegram home-assistant-component-tests.telegram_bot home-assistant-component-tests.tellduslive home-assistant-component-tests.template home-assistant-component-tests.tesla_wall_connector home-assistant-component-tests.threshold home-assistant-component-tests.tibber home-assistant-component-tests.tile home-assistant-component-tests.time_date home-assistant-component-tests.timer home-assistant-component-tests.tod home-assistant-component-tests.todoist home-assistant-component-tests.tolo home-assistant-component-tests.tomato home-assistant-component-tests.tomorrowio home-assistant-component-tests.toon home-assistant-component-tests.totalconnect home-assistant-component-tests.tplink home-assistant-component-tests.traccar home-assistant-component-tests.trace home-assistant-component-tests.tractive home-assistant-component-tests.tradfri home-assistant-component-tests.trafikverket_ferry home-assistant-component-tests.trafikverket_train home-assistant-component-tests.trafikverket_weatherstation home-assistant-component-tests.transmission home-assistant-component-tests.transport_nsw home-assistant-component-tests.trend home-assistant-component-tests.tts home-assistant-component-tests.tuya home-assistant-component-tests.twentemilieu home-assistant-component-tests.twilio home-assistant-component-tests.twinkly home-assistant-component-tests.twitch home-assistant-component-tests.uk_transport home-assistant-component-tests.ukraine_alarm home-assistant-component-tests.unifi home-assistant-component-tests.unifi_direct home-assistant-component-tests.unifiprotect home-assistant-component-tests.universal home-assistant-component-tests.upb home-assistant-component-tests.upcloud home-assistant-component-tests.update home-assistant-component-tests.upnp home-assistant-component-tests.uptime home-assistant-component-tests.uptimerobot home-assistant-component-tests.usb home-assistant-component-tests.usgs_earthquakes_feed home-assistant-component-tests.utility_meter home-assistant-component-tests.uvc home-assistant-component-tests.vacuum home-assistant-component-tests.vallox home-assistant-component-tests.velbus home-assistant-component-tests.venstar home-assistant-component-tests.vera home-assistant-component-tests.verisure home-assistant-component-tests.version home-assistant-component-tests.vesync home-assistant-component-tests.vicare home-assistant-component-tests.vilfo home-assistant-component-tests.vizio home-assistant-component-tests.vlc_telnet home-assistant-component-tests.voicerss home-assistant-component-tests.volumio home-assistant-component-tests.vulcan home-assistant-component-tests.vultr home-assistant-component-tests.wake_on_lan home-assistant-component-tests.wallbox home-assistant-component-tests.water_heater home-assistant-component-tests.watttime home-assistant-component-tests.waze_travel_time home-assistant-component-tests.weather home-assistant-component-tests.webhook home-assistant-component-tests.webostv home-assistant-component-tests.websocket_api home-assistant-component-tests.wemo home-assistant-component-tests.whirlpool home-assistant-component-tests.whois home-assistant-component-tests.wiffi home-assistant-component-tests.wilight home-assistant-component-tests.wiz home-assistant-component-tests.wled home-assistant-component-tests.workday home-assistant-component-tests.worldclock home-assistant-component-tests.ws66i home-assistant-component-tests.wsdot home-assistant-component-tests.xbox home-assistant-component-tests.xiaomi home-assistant-component-tests.xiaomi_aqara home-assistant-component-tests.xiaomi_miio home-assistant-component-tests.yale_smart_alarm home-assistant-component-tests.yamaha home-assistant-component-tests.yamaha_musiccast home-assistant-component-tests.yandex_transport home-assistant-component-tests.yandextts home-assistant-component-tests.yeelight home-assistant-component-tests.yolink home-assistant-component-tests.youless home-assistant-component-tests.zeroconf home-assistant-component-tests.zerproc home-assistant-component-tests.zha home-assistant-component-tests.zodiac home-assistant-component-tests.zone home-assistant-component-tests.zwave_js home-assistant-component-tests.zwave_me lollypop mailman mailman-hyperkitty manim mirage-im mopidy-scrobbler odoo opsdroid paperless-ngx pika-backup pithos platformio pylode python310Packages.aiocurrencylayer python310Packages.aiohttp-apispec python310Packages.aiosenz python310Packages.amcrest python310Packages.anyio python310Packages.asks python310Packages.asyncclick python310Packages.asyncwhois python310Packages.audible python310Packages.authcaptureproxy python310Packages.axis python310Packages.bimmer-connected python310Packages.bitcoinrpc python310Packages.cirq python310Packages.cirq-rigetti python310Packages.datasette python310Packages.datasette-template-sql python310Packages.denonavr python310Packages.dependency-injector python310Packages.devolo-plc-api python310Packages.elastic-apm python310Packages.elmax python310Packages.elmax-api python310Packages.entrance python310Packages.entrance-with-router-features python310Packages.envoy-reader python310Packages.etebase-server python310Packages.falcon python310Packages.fastapi python310Packages.fastapi-mail python310Packages.gidgethub python310Packages.glances-api python310Packages.ha-philipsjs python310Packages.httpcore python310Packages.httpx python310Packages.httpx-ntlm python310Packages.httpx-socks python310Packages.iotawattpy python310Packages.ipfshttpclient python310Packages.ircrobots python310Packages.ismartgate python310Packages.json-logging python310Packages.jupyter-lsp python310Packages.jupyter-server-mathjax python310Packages.jupyter_server python310Packages.jupyterlab python310Packages.jupyterlab-git python310Packages.jupyterlab-lsp python310Packages.jupyterlab_server python310Packages.jupytext python310Packages.libversion python310Packages.luftdaten python310Packages.nbclassic python310Packages.nbdime python310Packages.netdata python310Packages.onvif-zeep-async python310Packages.openapi-core python310Packages.ormar python310Packages.pycfmodel python310Packages.pyfuse3 python310Packages.pylast python310Packages.pylitterbot python310Packages.pypoint python310Packages.pypsrp python310Packages.pyquil python310Packages.pyrmvtransport python310Packages.pytest-aio python310Packages.pytest-httpx python310Packages.pytest-subprocess python310Packages.python-kasa python310Packages.qcs-api-client python310Packages.requests-aws4auth python310Packages.respx python310Packages.rmcl python310Packages.sanic python310Packages.sanic-auth python310Packages.sanic-testing python310Packages.simple-rest-client python310Packages.simple-salesforce python310Packages.slowapi python310Packages.starlette python310Packages.sunpy python310Packages.teslajsonpy python310Packages.timetagger python310Packages.total-connect-client python310Packages.uvicorn python310Packages.watchfiles python310Packages.watchgod python310Packages.webargs python310Packages.whodap python310Packages.youtube-search-python python310Packages.zeep python310Packages.zeversolarlocal python39Packages.aiocurrencylayer python39Packages.aiohttp-apispec python39Packages.aiosenz python39Packages.amcrest python39Packages.anyio python39Packages.asks python39Packages.asyncclick python39Packages.asyncwhois python39Packages.audible python39Packages.authcaptureproxy python39Packages.axis python39Packages.bimmer-connected python39Packages.bitcoinrpc python39Packages.cirq python39Packages.cirq-rigetti python39Packages.datasette-template-sql python39Packages.denonavr python39Packages.dependency-injector python39Packages.devolo-plc-api python39Packages.elastic-apm python39Packages.elmax python39Packages.elmax-api python39Packages.entrance python39Packages.entrance-with-router-features python39Packages.envoy-reader python39Packages.falcon python39Packages.fastapi python39Packages.fastapi-mail python39Packages.gidgethub python39Packages.glances-api python39Packages.ha-philipsjs python39Packages.httpcore python39Packages.httpx python39Packages.httpx-ntlm python39Packages.httpx-socks python39Packages.iotawattpy python39Packages.ipfshttpclient python39Packages.ircrobots python39Packages.ismartgate python39Packages.json-logging python39Packages.jupyter-lsp python39Packages.jupyter-server-mathjax python39Packages.jupyter_server python39Packages.jupyterlab python39Packages.jupyterlab-git python39Packages.jupyterlab-lsp python39Packages.jupyterlab_server python39Packages.jupytext python39Packages.libversion python39Packages.luftdaten python39Packages.nbclassic python39Packages.nbdime python39Packages.netdata python39Packages.onvif-zeep-async python39Packages.openapi-core python39Packages.ormar python39Packages.pycfmodel python39Packages.pyfuse3 python39Packages.pylast python39Packages.pylitterbot python39Packages.pypoint python39Packages.pypsrp python39Packages.pyquil python39Packages.pyrmvtransport python39Packages.pytest-aio python39Packages.pytest-httpx python39Packages.pytest-subprocess python39Packages.python-kasa python39Packages.qcs-api-client python39Packages.requests-aws4auth python39Packages.respx python39Packages.rmcl python39Packages.sanic python39Packages.sanic-auth python39Packages.sanic-testing python39Packages.simple-rest-client python39Packages.simple-salesforce python39Packages.slowapi python39Packages.starlette python39Packages.sunpy python39Packages.teslajsonpy python39Packages.timetagger python39Packages.total-connect-client python39Packages.uvicorn python39Packages.watchfiles python39Packages.watchgod python39Packages.webargs python39Packages.whodap python39Packages.youtube-search-python python39Packages.zeep python39Packages.zeversolarlocal s3ql soco-cli tauon theharvester timetagger tribler vorta wapiti xonsh ytmdl
```

**Pull request merged**